### ISSUE TITLE
option to hide join/part/quit messages from the chat window

### ISSUE THREAD

**Issue by gdamjan** [communi/communi-desktop#113]
I don't need to see parts and joins in the chat pane.

konversation handles this nicely, it only shows parts for people that were recently active (last 5 minutes).

**Comment by Venemo** [comment]
@gdamjan Right now we group together multiple join / part / quit messages, so that should reduce the noise. If I understand correctly you don't actually want to hide all of them, just want to show them based on user activity. Do I understand correctly?

**Comment by gdamjan** [comment]
@Venemo right, if a user was recently active I'm concerned about them leaving the chat,
 if not, I don't care at all.

**Comment by Venemo** [comment]
This should be solved together with #95
